### DISCUSSION TITLE
Homework 1; Part 3 - Question 1

### DISCUSSION THREAD

**Discussion by jmcdonnell876** [cooperrc/computational-mechanics#169] [Q&A]
I am unsure if I may have missed some available material on this matter however I have been struggling to determine what the equation is for the Analytical Solution of the problem. 

I originally assumed we would use the average mean between 1900 and 2020 multiplied by t added to p(0) but I now realize this is the measured data. Could someone possibly point me in the right direction for correctly modeling the analytical equation.

Thanks!

**Comment by cooperrc** [comment]
The analytical solution is the solution to 

$\frac{dP}{dt} = kP$
where $P(1900) = pop_0$

is

$P(t) = pop_0 e^{k(t-1900)}$
